### PR TITLE
GEOT-5150: bug FeatureTypeInfoImpl.getAbstract() on empty abstract list

### PR DESCRIPTION
**Pull request by janvenstermans**
https://osgeo-org.atlassian.net/browse/GEOT-5150

### CODE REVIEW

**Comment by aaime** [comment]
Substance is looking good, formally you forgot to update the licence header in FeatureTypeInfoImpl to state last change is 2015, and to add the licence header to the test class, using 2015 as the sole year.

File: modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/v2_0/FeatureTypeInfoImpl.java [hunk 1]
```diff
@@ -194,7 +194,9 @@ public String getAbstract() {
             sb.append(a);
             sb.append(" ");
         }
-        sb.setLength(sb.length()-1);
+		if (sb.length() > 0) {
+			sb.setLength(sb.length()-1);
+		}
         return sb.toString();
```
Comment (jodygarnett) [inline modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/v2_0/FeatureTypeInfoImpl.java]: got a bit confused between tabs and spaces on this one :)

File: modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/FeatureTypeInfoImplTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package org.geotools.data.wfs.internal.v2_0;
```
Comment (jodygarnett) [inline modules/unsupported/wfs-ng/src/test/java/org/geotools/data/wfs/internal/v2_0/FeatureTypeInfoImplTest.java]: please include project header when adding new files.

**Comment by jodygarnett** [comment]
rebased and merged to master

**Pull request closed without merging**